rosdep/base.yaml

```
@@ -8474,6 +8474,12 @@ rti-connext-dds-6.0.1:
   ubuntu:
     '*': [rti-connext-dds-6.0.1]
     bionic: null
+rti-connext-dds-7.3.0:
+  nixos: []
+  ubuntu:
+    '*': [rti-connext-dds-7.3.0]
```

```suggestion
    '*': [rti-connext-dds-7.3.0-ros]
```

Apologies for creating ambiguity here @lobolanja. The package is keeping the `-ros` suffix. My comments about dropping it only applied to the rosdep key name.

It's probably best just to keep it everywhere and follow convention. Sorry for the confusion.

---

@nuclearsandwich @mjcarroll   I think we can go with that approach—keeping the `-ros` suffix only for the package that’s installed via the .deb, and using `rti-connext-dds-7.3.0` as the key and in the package.xml.

At the end of the day, the real dependency is Connext 7.3.0, so the `-ros` suffix doesn’t really matter from a functionality standpoint, it is just to differentiate the package from RTI regular Debian packages.